advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx

```diff
@@ -1,24 +1,22 @@
 ---
-title: 'Implementing CipherTrust Transparent Encryption (CTE)'
+title: '3. Implementing CipherTrust Transparent Encryption (CTE)'
 description: 'A walk-through of setting up CTE'
 ---
 
-Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:
+**Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:**
 
-1. Postgres Server installed and in operation.
+  1. Postgres Server installed and operational.
```

Comment:
Postgres Server is a little ambiguous considering the three flavors of Postgres and that one of them was traditionally called Postgres Advanced Server. I think we may want to refer to server generically by using lower case, if what we mean is the Postgres server for whatever flavor of Postgres you are using.

Reply:
That said, I'm not sure you need to talk about the components here as they are shown in the previous architecture diagram and discussed in the following Prerequisites section.